feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -124,12 +124,11 @@ private const val IMAGE_CAPTURE_TRACE = "JCA Image Capture"
 class PreviewViewModel @Inject constructor(
     private val cameraSystem: CameraSystem,
     private val savedStateHandle: SavedStateHandle,
+    private val saveMode: SaveMode,
```

Comment:
With changes suggested in AppModule, this would become `val @DefaultSaveMode defaultSaveMode: SaveMode`.